commit 6435e5f65538
Author: Thomas Nguyen <tnguyen@mozilla.com>
Date:   Thu Sep 21 16:39:36 2017 +0800

    Bug 1356854 - Check mFD is not null before PR_Write. r=mcmanus, a=ritu
    
    MozReview-Commit-ID: INoTUvPpPrs
    
    --HG--
    extra : source : 638f326530b719ce386bb5d1701fed4ae4604690
---
 netwerk/base/nsFileStreams.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git netwerk/base/nsFileStreams.cpp netwerk/base/nsFileStreams.cpp
index ac259c1c7e94..6be501456656 100644
--- netwerk/base/nsFileStreams.cpp
+++ netwerk/base/nsFileStreams.cpp
@@ -378,6 +378,9 @@ nsFileStreamBase::DoPendingOpen()
 
     case eOpened:
         MOZ_ASSERT(mFD);
+        if (NS_WARN_IF(!mFD)) {
+          return NS_ERROR_FAILURE;
+        }
         return NS_OK;
 
     case eClosed:
